### DISCUSSION TITLE
无法正确打开storage.db

### DISCUSSION THREAD

**Discussion by xixohoej** [toeverything/AFFiNE#6699] [Q&A]
我之前一直使用的旧版本，今天更新之后发现我一直使用工作区不见了。然后我用Listary找了下，找到了一个最大的storage.db，应该就是我一直在使用的那个工作区。我创建了一个新工作区，将该工作区的storage.db直接替换为了原先的storage.db，结果无法打开。我还尝试了很多其他办法，都无法起作用。有谁知道该如何解决吗？里面的笔记对我来说真的很重要。

**Comment by pengx17** [comment]
可否将db文件发给我吗？我看看能不能恢复回来。我的邮箱是 pengxiao at toeverything.info

**Reply by xixohoej** [reply]
好的，谢谢你，我这就发给你
